pallets/data-signer/src/mock.rs

```diff
@@ -0,0 +1,149 @@
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+// USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+use crate as data_signer;
+use bridge_types::{traits::OutboundChannel, SubNetworkId};
+use frame_support::{parameter_types, traits::Everything};
+use frame_system as system;
+use sp_core::H256;
+use sp_runtime::{
+    testing::Header,
+    traits::{BlakeTwo256, IdentityLookup},
+    transaction_validity::TransactionPriority,
+};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+    pub enum Test where
+        Block = Block,
+        NodeBlock = Block,
+        UncheckedExtrinsic = UncheckedExtrinsic,
+    {
+        System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+        DataSigner: data_signer::{Pallet, Call, Storage, Event<T>},
+    }
+);
+
+parameter_types! {
+    pub const BlockHashCount: u64 = 250;
+    pub const SS58Prefix: u8 = 42;
+    pub const TestUnsignedPriority: TransactionPriority = 100;
+    pub const TestUnsignedLongevity: u64 = 100;
+    pub const BridgeMaxPeers: u32 = 50;
+}
+
+pub type AccountId = u64;
+
+impl system::Config for Test {
+    type BaseCallFilter = Everything;
+    type BlockWeights = ();
+    type BlockLength = ();
+    type DbWeight = ();
+    type RuntimeOrigin = RuntimeOrigin;
+    type RuntimeCall = RuntimeCall;
+    type Index = u64;
+    type BlockNumber = u64;
+    type Hash = H256;
+    type Hashing = BlakeTwo256;
+    type AccountId = u64;
+    type Lookup = IdentityLookup<Self::AccountId>;
+    type Header = Header;
+    type RuntimeEvent = RuntimeEvent;
+    type BlockHashCount = BlockHashCount;
+    type Version = ();
+    type PalletInfo = PalletInfo;
+    type AccountData = ();
+    type OnNewAccount = ();
+    type OnKilledAccount = ();
+    type SystemWeightInfo = ();
+    type SS58Prefix = SS58Prefix;
+    type OnSetCode = ();
+    type MaxConsumers = frame_support::traits::ConstU32<16>;
+}
+
+impl data_signer::Config for Test {
+    type RuntimeEvent = RuntimeEvent;
+    type OutboundChannel = TestOutboundChannel;
+    type CallOrigin = TestCallOrigin;
+    type UnsignedPriority = TestUnsignedPriority;
+    type UnsignedLongevity = TestUnsignedLongevity;
+    type MaxPeers = BridgeMaxPeers;
+    type WeightInfo = ();
+}
+
+pub struct TestOutboundChannel;
+impl OutboundChannel<SubNetworkId, AccountId, ()> for TestOutboundChannel {
+    fn submit(
+        _network_id: SubNetworkId,
+        _who: &system::RawOrigin<AccountId>,
+        _payload: &[u8],
+        _additional: (),
+    ) -> Result<H256, sp_runtime::DispatchError> {
+        Ok([
+            1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+            1, 1, 1,
+        ]
+        .into())
```

Comment:
```suggestion
        Ok([1; 32]
        .into())
```
or
```suggestion
        Ok(H256::repeat_byte(1))
```
Here and further.